scripts/data_preprocessing/preparse_alignments_of3.py

```diff
@@ -64,8 +65,8 @@ def main(
     """Preparse multiple sequence alignments for AF3 dataset."""
     try:
         max_seq_counts = json.loads(max_seq_counts)
-    except json.JSONDecodeError:
-        click.echo("Invalid max_seq_counts JSON string!")
+    except json.JSONDecodeError as e:
+        raise click.ClickException(f"Invalid max_seq_counts JSON string: {e}")
```

Comment:
@vinay-swamy this is actually silently continuing when you provide invalid JSON, haha, pretty major footman